src/connectors/search-box/connectSearchBox.js

```diff
@@ -75,8 +77,22 @@ export default function connectSearchBox(renderFn) {
   return (widgetParams = {}) => {
     const {queryHook} = widgetParams;
 
+    function clear(helper) {
+      return function() {
+        helper
+          .setState(helper.state.setQuery(''))
```

Comment:
Nope, I've used the same code as https://github.com/algolia/instantsearch.js/blob/feat/instantsearch.js/v2/src/lib/utils.js#L222-L224